python/tk_houdini_renderman/handler.py

```diff
@@ -35,29 +37,20 @@ def __init__(self, app):
         self.sg = self.app.shotgun
 
     def submit_to_farm(self, node, network):
-        if not self.__validate_node(node, network):
-            return
+        """Start farm render
 
-        # Set paths and prepare node
+        :param node: RenderMan node
+        :param network: Network type
+        """
+        # Create directories
         render_name = node.parm("name").eval()
-        render_path = self.__calculate_path(node, network)
-        self.__apply_path(node, render_path, network)
-        self.__create_directory(render_path)
-
-        # Set filters
-        if network == "lop":
-            filters = self.__check_lop_filters(node)
-            self.__set_lop_filter_filename(node, filters)
-
-        if network == "rop":
-            filters = self.__check_rop_filters(node)
-            self.__set_rop_filter_filename(filters, render_node=node)
+        render_paths = node.type().hdaModule().get_render_paths()
+        for p in render_paths:
```

Comment:
Here I would suggest to use the name of the thing that is iterated over. So "p" would be "path"